docs/faq/users/anvil.rst

```diff
@@ -0,0 +1,227 @@
+Recipe: Run a Job on Anvil Using an Allocation
+----------------------------------------------
+
+This recipe assumes that you have decided to use your allocation
+for Anvil to run one of your HTCondor jobs.  It takes you step by
+step through the process of Bringing Your Own Resources (BYOR) in the
+form of an allocation to an OSG Connect access point and using that
+resource to run your HTCondor job.  In what follows, we refer to the
+named set of resources leased from that allocation as an *annex*.
+
+In this recipe, we assume that the job has not yet been placed at an
+OSG Connect access point when we begin.
+
+Ingredients
+===========
+
+- An OSG Connect account and password
+- An HTCondor job submit file (:doc:`example.submit <annex-example-job>`).
+- An allocation for Anvil.
+- Command-line login access to Anvil (see
+  `Purdue's instructions for gaining access <https://www.rcac.purdue.edu/knowledge/anvil/access>`_).
+  We'll use ``LOGIN_NAME`` to refer to your login name on Anvil.
+- A name for your Anvil annex (example).  By convention,
+  this is the name of the submit file you want to run, without its extension.
+
+Assumptions
+===========
+
+- You want to run the job described above on Anvil.
+- The job described above fits within the capabilities of the
+  `Anvil partition (queue) <https://www.rcac.purdue.edu/knowledge/anvil/run/partitions>`_
+  you wish to run it on.
+
+Preparation
+===========
+
+First, you will need to determine the project ID of your allocation on
+Anvil.  If you already know your project ID, you can skip this
+section.  If not, log in to ``anvil.rcac.purdue.edu`` in a terminal and run the
+following command.  (Don't copy the ``$``; in this and other examples
+further down the page, the ``$`` just signifies something you type in,
+rather than something that the computer prints out.)
+
+.. code-block:: text
+
+    $ mybalance
+
+Choose one of the rows in the top half of the table (there may be only
+one row) and remember the entry in the NAME column.
+For the rest of these instructions, we'll use ``PROJECT_ID`` where you
+need to put that entry.
+
+Instructions
+============
+
+1. Log into the OSG Connect Access Point
+''''''''''''''''''''''''''''''''''''''''
+
+Log into an OSG Connect access point (e.g., ``login04.osgconnect.net`` or
+``login05.osgconnect.net``) using your OSG Connect account and password.
+
+2. Submit the Job
+'''''''''''''''''
+
+Submit the job on the access point, indicating that you want it to run
+on your own resource (the Anvil allocation, in this case) with the
+``--annex-name`` option:
+
+.. code-block:: text
+
+    $ htcondor job submit example.submit --annex-name example
+    Job 123 was submitted and will run only on the annex 'example'.
+
+Notes on the output of this command:
```

Comment:
Should this be a `.. note:` note?

Reply:
I didn't think of that; I'll try it and let you know.

Reply:
It should be.  I'll push a patch along.